docs/search/full-text/index.mdx

```diff
@@ -295,7 +295,7 @@ SELECT * FROM search_idx.schema();
 </ParamField>
 <ParamField body="icu">
   Tokenizes text using the ICU tokenizer, which uses Unicode Text Segmentation and is suitable for tokenizing most
-  languages.
+  languages. Ensure `pg_search` is compiled with the optional icu dependency.
```

Comment:
I think we should remove this here. Most people reading the docs are not compiling themselves, and our pre-built versions compile with ICU

Reply:
ah alright